website/docs/add-secure-apps/providers/rac/how-to-rac.md

```diff
@@ -2,41 +2,42 @@
 title: Create a Remote Access Control (RAC) provider
 ---
 
-The RAC provider is a highly flexible feature for accessing remote machines. This document provides instructions for the basic creation and configuration of a RAC provider within a defined scenario.
+## Introduction
```

Comment:
I'd leave this out... most of our docs start with a few words about the topic below, it is understood to be the Intro section.